File: Folder PHP/Barang.php
<?php
class Barang {
    private int $id;
    private string $nama;
    private int $harga;
    private int $stok;

    public function __construct(int $id = 0, string $nama = "", int $harga = 0, int $stok = 0) {
        $this->id = $id;
        $this->nama = $nama;
        $this->harga = $harga;
        $this->stok = $stok;
    }

    public function getId(): int { return $this->id; }
    public function setId(int $id) { $this->id = $id; }

    public function getNama(): string { return $this->nama; }
    public function setNama(string $nama) { $this->nama = $nama; }

    public function getHarga(): int { return $this->harga; }
    public function setHarga(int $harga) { $this->harga = $harga; }

    public function getStok(): int { return $this->stok; }
    public function setStok(int $stok) { $this->stok = $stok; }
}

File: Python/Barang.py
class Barang:
    def __init__(self, id=0, nama="", harga=0, stok=0):
        self._id = id
        self._nama = nama
        self._harga = harga
        self._stok = stok

    # getters & setters
    @property
    def id(self): return self._id
    @id.setter
    def id(self, value): self._id = value

    @property
    def nama(self): return self._nama
    @nama.setter
    def nama(self, value): self._nama = value

    @property
    def harga(self): return self._harga
    @harga.setter
    def harga(self, value): self._harga = value

    @property
    def stok(self): return self._stok
    @stok.setter
    def stok(self, value): self._stok = value
